src/Solution/ParticleTracker/Particle.f90

```diff
@@ -285,9 +307,17 @@ subroutine save_particle(this, particle, ip)
     this%extend(ip) = particle%iextend
   end subroutine save_particle
 
-  !> @brief Apply the given global-to-local transformation to the particle.
+  !> @brief Transform particle coordinates.
+  !!
+  !! Apply a translation and/or rotation to particle coordinates.
+  !! No rescaling. It's also possible to invert a transformation.
+  !!
+  !! Upon inverting, be sure to untransform to eliminate roundoff
+  !! error accumulated in the particle origin/rotation variables,
+  !! and unsetting the transformed flag.
+  !<
```

Comment:
not sure i understand "untransform to eliminate roundoff" - not saying it's wrong, just don't remember and don't immediately see it